component/booksCard.jsx

```diff
@@ -0,0 +1,54 @@
+import useStore from '../hooks/useStore';
+import { useEffect } from 'react';
+import styled from 'styled-components';
+
+const Api = process.env.NEXT_PUBLIC_REACT_APP_API_KEY;
+
+export default function BooksCard({ apikey }) {
+  const fetchApi = useStore(state => state.fetchApi);
+  const fetchedData = useStore(state => state.fetchedData);
+
+  useEffect(() => {
+    fetchApi(
+      `https://api.nytimes.com/svc/books/v3/lists/current/hardcover-fiction.json?api-key=${apikey}`
+    );
+  }, [fetchApi]);
+  console.log(Api);
+  return (
+    <CardWrapper>
+      {fetchedData.results.books !== undefined ? (
+        fetchedData.results.books.map((book, index) => {
+          return (
+            <StyledCard>
+              <article key={index}>
+                <img src={book.book_image}></img>
+                <h3>{book.title}</h3>
+                <p>{book.author}</p>
+                <p>{book.description}</p>
+              </article>
+            </StyledCard>
```

Comment:
you could actually outsource this part into its own component. But you can do this in a later user-story :)